pkgs/by-name/we/weblate/package.nix

```diff
@@ -39,15 +40,15 @@ python.pkgs.buildPythonApplication rec {
     owner = "WeblateOrg";
     repo = "weblate";
     tag = "weblate-${version}";
-    hash = "sha256-/fsNQvIIgcTPZHHIwr8sruEJpPJTmXbevoxy1GPmOOU=";
+    hash = "sha256-DRodQb4IvLfpL+TzLigtiKTvXvGYbpa9Ej4+fCHSGmo=";
   };
 
   patches = [
     # FIXME This shouldn't be necessary and probably has to do with some dependency mismatch.
     ./cache.lock.patch
   ];
 
-  build-system = with python.pkgs; [ setuptools ];
+  build-system = [ gettext ] ++ (with python.pkgs; [ setuptools ]);
```

Comment:
gettext should go into nativeBuildInputs intead. build-system is specifically for that setting in pyproject.toml or comparable